app/views/stations/index.erb

```diff
@@ -1,20 +1,26 @@
 <%= erb :banner %>
-<head><title>All Stations</title></head>
 
-<nav class="crumbs">
-  <a href="/">Home</a> /
-  Stations
-</nav>
+<head>
+  <title>All Stations</title>
+</head>
 
-<a class="new_button" href="/stations/new">New Station</a>
-<!-- build out breadcrumb path to replace above -->
+<ul class="breadcrumb">
+  <li><a href="/">Home</a></li>
+  <li>Stations</li>
+</ul>
+
+<a href="/stations/new"><button class="button" type="button">New Station</button></a>
```

Comment:
Capybara was having trouble testing the button. I'll push an update that changes this back to a link with class="button" with additional properties defined so that it looks like a button. @Ecksi